editor/property_selector.cpp

```diff
@@ -40,32 +41,6 @@ void PropertySelector::_text_changed(const String &p_newtext) {
 	_update_search();
 }
 
-void PropertySelector::_sbox_input(const Ref<InputEvent> &p_event) {
-	// Redirect navigational key events to the tree.
-	Ref<InputEventKey> key = p_event;
-	if (key.is_valid()) {
-		if (key->is_action("ui_up", true) || key->is_action("ui_down", true) || key->is_action("ui_page_up") || key->is_action("ui_page_down")) {
-			search_options->gui_input(key);
-			search_box->accept_event();
-
-			TreeItem *root = search_options->get_root();
-			if (!root->get_first_child()) {
-				return;
-			}
-
-			TreeItem *current = search_options->get_selected();
-
-			TreeItem *item = search_options->get_next_selected(root);
-			while (item) {
-				item->deselect(0);
-				item = search_options->get_next_selected(item);
-			}
-
-			current->select(0);
```

Comment:
No idea what the extra code was for. Removing it doesn't seem to have changed anything.